BookingsAPI/Bookings.API/Controllers/PersonsController.cs

```diff
@@ -133,13 +133,13 @@ public async Task<IActionResult> GetPersonSuitabilityAnswers(string username)
         private PersonSuitabilityAnswerResponse BuildResponse(VideoHearing hearing, string username)
         {
             var participant = hearing.Participants.FirstOrDefault(p => p.Person.Username.ToLower() == username.Trim().ToLower());
-            var answers = participant.SuitabilityAnswers;
+            var answers = participant.Questionnaire != null ? participant.Questionnaire.SuitabilityAnswers: null;
```

Comment:
always use empty lists rather than null values

Reply:
Changed to list